service/controllers/messages.go

```diff
@@ -98,66 +98,104 @@ func routes(routeObject response, w http.ResponseWriter) {
-			if strings.ToLower(thirdPars) == "for" || strings.ToLower(thirdPars) =="videos" {
-				strArr0=firstPars+" "+secondPars+" "+thirdPars
-				strArr=strings.Split(strArr0," ")
-				remainingString=strings.Join(stringDifference(strArr,strArrDiff)," ")
-				fmt.Println(remainingString)
-			} else {
-				strArr0=firstPars+" "+secondPars
-				strArr=strings.Split(strArr0," ")
-				remainingString=strings.Join(stringDifference(strArr,strArrDiff)," ")
-				fmt.Println(remainingString)
-			}
-		} else {
-				strArr = strings.Split(firstPars, " ")
-				strArrDiff = strings.Split(message, " ")
-				remainingString=strings.Join(stringDifference(strArr,strArrDiff)," ")
-				fmt.Println(remainingString)
+	// message := routeObject.message
+	// messageArr := strings.Split(message, " ")
+	// var matchPars string
+	// var secondPars string
+	// var thirdPars string
+	// if len(messageArr)>=3 {
+	// 	matchPars=messageArr[0]
+	// 	secondPars = messageArr[1]
+	// 	thirdPars = messageArr[2]
+	// } else if len(messageArr)==2 {
+	// 	matchPars=messageArr[0]
+	// 	secondPars = messageArr[1]
+	// } else if(len(messageArr)==1) {
+	// 		matchPars=messageArr[0]
+	// }
+
+	// var remainingString string
+	// var strArr []string
+	// var strArr0 string
+	// var strArrDiff []string
+	// strArrDiff = strings.Split(message, " ")
+	// if strings.ToLower(matchPars) == "google" || strings.ToLower(matchPars) =="yahoo" || strings.ToLower(matchPars) =="bing" || strings.ToLower(matchPars) =="search" || strings.ToLower(matchPars) =="youtube"|| strings.ToLower(matchPars) == "watch"||strings.ToLower(matchPars) =="videos" || strings.ToLower(matchPars) =="images" || strings.ToLower(matchPars) =="image" || strings.ToLower(matchPars) =="meaning"{
+	// 	if strings.ToLower(secondPars) == "search" || strings.ToLower(secondPars) =="google" ||strings.ToLower(secondPars) =="yahoo" ||strings.ToLower(secondPars) =="bing" ||strings.ToLower(secondPars) =="videos" ||strings.ToLower(secondPars) =="youtube"||strings.ToLower(secondPars) =="for" || strings.ToLower(secondPars) =="of"{
+	// 		if strings.ToLower(thirdPars) == "for" || strings.ToLower(thirdPars) =="videos" {
+	// 			strArr0=matchPars+" "+secondPars+" "+thirdPars
+	// 			strArr=strings.Split(strArr0," ")
+	// 			remainingString=strings.Join(stringDifference(strArr,strArrDiff)," ")
+	// 			fmt.Println(remainingString)
+	// 		} else {
+	// 			strArr0=matchPars+" "+secondPars
+	// 			strArr=strings.Split(strArr0," ")
+	// 			remainingString=strings.Join(stringDifference(strArr,strArrDiff)," ")
+	// 			fmt.Println(remainingString)
+	// 		}
+	// 	} else {
+	// 			strArr = strings.Split(matchPars, " ")
+	// 			strArrDiff = strings.Split(message, " ")
+	// 			remainingString=strings.Join(stringDifference(strArr,strArrDiff)," ")
+	// 			fmt.Println(remainingString)
+	// 		}
```

Comment:
please remove these comments. I have reviewed the code and looks cool!

Reply:
Yaa...Thank you.